samples/client/petstore/groovy/bin/main/org/openapitools/api/ApiUtils.groovy

```diff
@@ -0,0 +1,54 @@
+package org.openapitools.api
+
+import static groovyx.net.http.HttpBuilder.configure
+import static java.net.URI.create
+
+class ApiUtils {
```

Comment:
I guess the groovy files are unrelated to this PR?

Reply:
for some reason building the projects creates all these groovy files, idk what is going on